content/en/docs/tasks/traffic-management/ingress/secure-ingress/index.md

```diff
@@ -344,8 +344,8 @@ its clients, and we must use the name `cacert` to hold the CA certificate.
 
 {{< text bash >}}
 $ kubectl -n istio-system delete secret httpbin-credential
-$ kubectl create -n istio-system secret generic httpbin-credential --from-file=tls.key=httpbin.example.com.key \
---from-file=tls.crt=httpbin.example.com.crt --from-file=ca.crt=example.com.crt
+$ kubectl create -n istio-system secret generic httpbin-credential --from-file=key=httpbin.example.com.key \
+--from-file=cert=httpbin.example.com.crt --from-file=cacert=example.com.crt
```

Comment:
If these keys were wrong, why has the test not been failing? https://github.com/istio/istio.io/blob/33125be32a46cc164abe143fb7d936e652f0b237/content/en/docs/tasks/traffic-management/ingress/secure-ingress/test.sh#L93

Reply:
agree @frankbu I'll close the PR for now and look for more details.